### PR TITLE
19 Update the lesson Condiciones (TWP10)

### PR DESCRIPTION
**Pull request by y-martinez**
## Summary

This PR close y-martinez/PyZombis#19 

- This lesson has problems with its images, the links were broken. Now it works.
- According to the video playlist of the original course in its 2.0 version, the lesson had other things that were not yet in PyZombis. Now, they were added.
- Change the ``id`` following the rule.

## Checklist

- [x] Variables, functions and comments are translated to Spanish
- [x] Functions follow underscore notation
- [x] Spell check done & typos fixed
- [x] All python code is PEP8 compliant
- [x] Test coverage with Playwright implemented; locators are Pyhton code
- [x] Reviewers assigned (all peers & at least 1 mentor)

### CODE REVIEW

**Review by LeoCumpli21 — APPROVED** [review]
Everything seems to work just fine! LGTM

**Review by NicolasSandoval — COMMENTED** [review]
Great work! I leave you some questions:

- In the title 'else', the examples should not be 'else'?
- the text 'if/ else', should not be before? Or is it at the end because it is a summary of what has been seen before?
- the sentence at the end 'list of exercises "again"' what exercises does it refer to?

**Comment by y-martinez** [comment]
> Great work! I leave you some questions:
> 
> * In the title 'else', the examples should not be 'else'?
> * the text 'if/ else', should not be before? Or is it at the end because it is a summary of what has been seen before?
> * the sentence at the end 'list of exercises "again"' what exercises does it refer to?

Yeah, the reason is that the slides from the original course have the titles like that, however, I agree with you, I can change them to maintain a logical order. All the lessons have at the bottom the title "Lista de ejercicios again". I saw the videos from the original course and professor Massanori link in this section to some of the exercises we put into the Quizzes, maybe a good change should be to link them.

File: _sources/lectures/TWP10.rst [hunk 1]
```diff
@@ -3,273 +3,225 @@ Condiciones
 ===========
 
 
-.. image:: img/TWP10_001.jpeg
-   :height: 15.125cm
-   :width: 9.382cm
-   :align: center
-   :alt: 
+..  image:: img/TWP10_001.jpeg
```
Comment (angelasofiaremolinagutierrez) [inline _sources/lectures/TWP10.rst:6]:
```suggestion
.. image:: img/TWP10_001.jpeg
```

File: _sources/lectures/TWP10.rst [hunk 2]
```diff
@@ -3,273 +3,225 @@ Condiciones
...
+    :alt:
 
 
 if
 ==
 
-
 + Lea dos valores enteros e imprima el más grande
 
+..  codelens:: cl_l10_2
```
Comment (angelasofiaremolinagutierrez) [inline _sources/lectures/TWP10.rst:52]:
```suggestion
.. codelens:: cl_l10_2
```

**Review by angelasofiaremolinagutierrez — COMMENTED** [review]
Just one question. Is there a reason on why you put two spaces after the dots for a directive?
Is it because of the formatter tool? The syntax only contemplates one space.

> image
 https://docutils.sourceforge.io/docs/ref/rst/directives.html

**Review by NicolasSandoval — APPROVED** [review]
LGTM!

**Pull request merged**